go-controller/pkg/ovn/master.go

```diff
@@ -416,6 +418,51 @@ func (oc *DefaultNetworkController) addNode(node *kapi.Node) ([]*net.IPNet, erro
 		return nil, err
 	}
 
+	// For live migrated pods the switch manager has to be refill in
+	// case the node has take over the vm subnet
+	liveMigratablePods, err := kubevirt.FindLiveMigratablePods(oc.watchFactory)
+	if err != nil {
+		return nil, err
+	}
+	for _, liveMigratablePod := range liveMigratablePods {
```

Comment:
I think it would be interesting to do this only when the subnet annotaiton of a node changes.

To better do this is could be handled in `addUpdateLocalNodeEvent` after addNode is called.